### ISSUE TITLE
Add support for groundspeak:personal_note

### ISSUE THREAD

**Issue by GoogleCodeExporter** [opencaching/okapi#294]
```
By boguslaw.szczepanowski from 
https://code.google.com/p/opencaching-api/issues/detail?id=40#c13

Is it possible to change generated GPX file to include user notes in a separate 
XML tag, named groundspeak:personal_note?

(for the one-line summary see bottom of this comment)

That would require the following changes:
1. Upgrade groundspeak schema to http://www.groundspeak.com/cache/1/0/2 
2. Validate generated GPX/xml against new schema. There are several changes in 
attribute types and mandatory vs optional flag
3. Change my_notes parameter, so 
a) it could accept additional parameter value, called gc:personal_note, which 
will generate those previously mentioned tag
b) the parameter should accept a pipe separated list, so the user notes can be 
included both in a cache description, as well as in groundspeak:personal_note 
tag

We could deal with the schema change in three different ways:
1. Upgrade the schema to http://www.groundspeak.com/cache/1/0/2, and change GPX 
generation, so it will not violate the new schema
2. Upgrade the schema, but violate it and don't change GPX xml generation
3. Violate old schema (http://www.groundspeak.com/cache/1/0/1), and include tag 
groundspeak:personal_note even though the schema does not allow it

The first solution, although does not violate spec, seems to be the worst. 
First, some attribute types changed from xs:string to xs:boolean, which will 
cause case changes (from True to true, and False to false). Other attribute 
types changed from xs:string to xs:dateTime. The software may parse it 
case-sensitive, or other specific way, so our change my introduce 
incompatibilities. Second, it's most work-intensive change for us to implement 
and test.

The third solution seem to be the most backward compatible, since most software 
does not care about unknown tags and simply ignores them. Some software may 
still pay attention to the schema URL (i.e.: Awaryjniejszy GPX does this), 
which renders the third solution to be better than the second one.

The new schema XSD, as well as changes description can be found here: 
https://docs.google.com/file/d/0BzM5DAJdpBIgMjRjZDA5OTctZjkzMS00ZDk5LWFmM2YtY2Ex
ODJjZTA0MmVm/edit?pli=1

Locus support is described here: 
https://getsatisfaction.com/locus/topics/export_notes_associated_to_geocaches#re
ply_9062574

Some other info here:
http://forum.locusmap.eu/index.php?PHPSESSID=tpm9707baltdpp8kgkgvlatbd1&topic=34
88.msg25219#msg25219


So in summary, my opinion is:
1. add gc:personal_note parameter value
2. generate groundspeak:personal_note tag
3. don't change any other XML-related stuff
```

Original issue reported on code.google.com by `rygielski` on 19 Dec 2013 at 7:58
